models/vrp.rb

```diff
@@ -57,6 +55,10 @@ def self.create(hash, options = {})
       }.merge(hash[:configuration] || {})
       Models.delete_all if options[:delete]
 
+      hash[:vehicles] ||= []
+      hash[:services] ||= []
+      hash[:points] ||= []
```

Comment:
I thought fields of type `has_many` are automatically initialized with an empty array? Is that not the case?

Reply:
The problem comes from the multiple validation/filter/transformations

Reply:
Oh yeah, the filters are based on the fact that grape validates that there are at least these members. I agree the easiest is what you already suggested :+1: